configured tap and schema

prakharcode opened the pull request.

edgarrmondragon commented on tap_kustomer/streams.py:
```diff
@@ -56,6 +56,124 @@ def get_child_context(
+        context: dict | None = None,  # noqa: ARG002
+    ) -> dict | None:
+        if isinstance(
+            row["attributes"].get("firstMessageIn", {}).get("meta", {}).get("to"),
```
@prakharcode Can you refactor this so the nested field is looked up only once.

e.g.

```python
def post_process(
    self,
    row: dict,
    context: dict | None = None,  # noqa: ARG002
) -> dict | None:
    first_message_to = row["attributes"].get("firstMessageIn", {}).get("meta", {}).get("to")
   first_message_cc = ...
    last_message_to = ...
    if isinstance(message_to, dict):
        ...
```

edgarrmondragon reviewed (commented):
Thanks @prakharcode. I left a suggestion, let me know if it makes sense.

prakharcode commented:
I did a few more changes -- including the refacrtor.

- Introduced flattening
- removed required keyword because it was not necessary
- few more changes which are relevant
